src/shared/services/commandInterpreterHelper.js

```diff
@@ -384,18 +384,17 @@ const availableCommands = [
     exec: (action, cmdchar, put, store) => {
       const state = store.getState()
 
-      // Since we now also handle queries with the :auto prefix, we need to strip that
-      // and attach to the actions object
-      const query = action.cmd.replace(
-        getCmdChar(state) + autoCommitTxCommand,
-        ''
-      )
-      action.query = query.trim()
+      // Since we now also handle queries with the :auto prefix,
+      // we need to strip that and attach to the actions object
+      const query = action.cmd.trim()
```

Comment:
This should strip any comments as well.

```
// Comment
:auto CREATE (n:Test {val: 'yo:auto'}) RETURN *
```
will now fail

Reply:
Good catch!

Reply:
I updated the implementation and added some tests